core/jazz_metrics/components/utils.js

```diff
@@ -240,9 +240,26 @@ function updateCloudfrontAsset(newAssetObj, relativeId) {
   return newAssetObj;
 }
 
+function initCloudWatch() {
+  var cloudwatch = new AWS.CloudWatch({
+    apiVersion: '2010-08-01'
+  });
+  return cloudwatch;
+}
+
+function cfCloudWatch() {
+  var cfcloudwatch = new AWS.CloudWatch({
+    apiVersion: '2010-08-01',
+    region: global_config.CF_REGION
```

Comment:
I see that the value for CF_REGION is hard coded in global_config. It was a miss on the earlier review for metrics, so can you fix it?

Reply:
Since cloudFront region (CF_REGION) is always 'us-east-1', it is hard coded in global_config.